README.md

```diff
@@ -109,13 +109,13 @@ Server](https://discord.gg/mMRrZQW) or [Reddit Community](https://www.reddit.com
 
 ## Contributing
 
-Contributions to Manim are always welcome. In particular, there is a dire need for tests and documentation. For contribution guidelines, please see the [documentation](https://docs.manim.community/en/latest/contributing.html).
+Contributions to Manim are always welcome. In particular, there is a dire need for tests and documentation. For contribution guidelines, please see the [documentation](https://docs.manim.community/en/stable/contributing.html).
 
-Most developers on the project use [Poetry](https://python-poetry.org/docs/) for management. You'll want to have poetry installed and available in your environment. You can learn more `poetry` and how to use it at its [documentation](https://docs.manim.community/en/latest/installation/for_dev.html).
+Most developers on the project use [Poetry](https://python-poetry.org/docs/) for management. You'll want to have poetry installed and available in your environment. You can learn more `poetry` and how to use it at its [documentation](https://docs.manim.community/en/stable/installation/for_dev.html).
 
 ## Code of Conduct
 
-Our full code of conduct, and how we enforce it, can be read on [our website](https://docs.manim.community/en/latest/conduct.html).
+Our full code of conduct, and how we enforce it, can be read on [our website](https://docs.manim.community/en/stable/conduct.html).
```

Comment:
I think this can be latest, but I'm not sure. I don't expect the CoC to change much.

Reply:
Thinking about it, changes to the CoC will probably only happen in response to a serious incident that would warrant making the change happen with immediate effect, so I'll change it to latest.

Reply:
Actually, thinking about it _again_, I disagree with changing any of the links in README back to stable. The rationale is that a curious user might click any of those links, check out the documentation and be in the wrong version. I think it's best to eliminate direct links to the latest version as much as possible.

Reply:
We want users to read the latest Code of Conduct/Contributing. If they're reading an the older version -- they may not be reading the same CoC/Contributing sections and thereby might not end up contributing with as much knowledge as they can be. #1130 and #1137 are examples of the contributing guidelines changing that might not be merged before v0.5.0. We want contributors to have as much information as they can before contributing.

> a curious user might click any of those links, check out the documentation and be in the wrong version

There isn't a wrong version, just one that doesn't match the user's current installed version. I think the solution should be to make the version of the documentation they're using more apparent. Maybe that's outside of the scope of this PR, but it's something to be addressed.

Reply:
I agree that we want users to read the latest CoC and contributing guidelines. However, for the case of contributing, we can generally expect that potential contributors are relatively familiar with the library and the docs, and can switch to the latest version. Moreover, we can add a specific note to the contribution guidelines page asking contributors to make sure they're reading the latest version. The amount of users is always going to be far more than the amount of contributors, so I think it makes sense to link to the stable version, given that we can always explicitly ask contributors to read the latest version anyways.

As for the Code of Conduct, it is very unlikely that there will be changes for it, and will likely only happen in a major incident. In that case, changing the README to point to the latest version can be done on the same PR as the change to the Code of Conduct.

There are definitely some disadvantages to linking to the stable version in these two cases, but I think this is outweighed by the potential save in time/headache for new users (who will far outnumber contributors) who may not be familiar with the versioning system of the docs.